venues/ICML.cc/2017/RML/webfield/conf.html

```diff
@@ -0,0 +1,97 @@
+
+// ------------------------------------
+// Basic venue homepage template
+//
+// This webfield displays the conference header (#header), the submit button (#invitation),
+// and a list of all submitted papers (#notes).
+// ------------------------------------
+
+// Constants
+var CONFERENCE = "ICML.cc/2017/RML";
+var INVITATION = CONFERENCE + '/-/Submission';
+var SUBJECT_AREAS = [
+  // Add conference specific subject areas here
+];
+var BUFFER = 1000 * 60 * 30;  // 30 minutes
+var PAGE_SIZE = 50;
+
+var paperDisplayOptions = {
+  pdfLink: true,
+  replyCount: true,
+  showContents: true
+};
+
+// Main is the entry point to the webfield code and runs everything
+function main() {
+  Webfield.ui.setup('#group-container', CONFERENCE);  // required
+
+  renderConferenceHeader();
+
+  load().then(render).then(function() {
+    Webfield.setupAutoLoading(INVITATION, PAGE_SIZE, paperDisplayOptions);
+  });
+}
+
+// RenderConferenceHeader renders the static info at the top of the page. Since that content
+// never changes, put it in its own function
+function renderConferenceHeader() {
+  Webfield.ui.venueHeader({
+    title: "ICML 2017 RML Workshop",
+    subtitle: "Reproducibility in Machine Learning",
+    location: "Syndey, Autralia",
+    date: "June 17th, 2017 5:00pm",
+    website: "https://2017.icml.cc/Conferences/2017/Schedule?showEvent=16",
+    instructions: null,  // Add any custom instructions here. Accepts HTML
+//    deadline: '2017-06-06 17:00:00'
```

Comment:
I'm surprised to see that.  It appears it came from my admin/conference-template file.  It was only supposed to be changed for the RSS conference. I'll look into it.